src/Table/Table.stories.jsx

```diff
@@ -46,6 +46,17 @@ const catColumns = [
   },
 ];
 
+const sort = function sort(firstElement, secondElement, key, direction) {
```

Comment:
This is just a utility function to sort catData by key and in a particular direction, since Javascript's Arrays.sort() function doesn't support sorting objects, obviously.